vg/vggio/context.go

```diff
@@ -0,0 +1,85 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package vggio // import "gonum.org/v1/plot/vg/vggio"
+
+import (
+	"image"
+	"image/color"
+
+	"gioui.org/f32"
+	"gioui.org/op"
+	"gioui.org/op/clip"
+
+	"gonum.org/v1/plot/vg"
+)
+
+type context struct {
+	color   color.Color // color is the current color.
+	linew   vg.Length   // linew is the current line width.
+	pattern []vg.Length // pattern is the current line style.
+	offset  vg.Length   // offset is the current line style.
+	stk     op.StackOp  // stk is the Gio op context stack.
+}
+
+type ctxops struct {
```

Comment:
It would be nice to have some documentation on this and its fields and methods.

Please place it above `context`.